z.lua

```diff
@@ -1901,6 +1925,7 @@ function z_shell_init(opts)
 			end
 		end
 		print(script_complete_bash)
+		print(script_fzf_complete_bash)
```

Comment:
> It will override the default completion function (which is crap I know).

Yes, with bash it's much better.

> zsh completion in z.lua is very handy, I don't want to it get overrided by fzf by default.

The PR only define it for bash. I don't know how to define complete option for zsh

Reply:
done in 7f1dd0a and bc0403e